traveller/modules/box__default/auth/models.py

```diff
@@ -102,6 +104,13 @@ def password(self, plaintext):
 
     def check_password(self, password):
         return check_password_hash(self._password, password)
+    
+    def has_role(self, role_):
+        for r_ in self.roles:
+            if r_.name == role_:
+                return True
+        return False
```

Comment:
why all those weird underscores?

Reply:
They are so weird, but can be replaced by non weird sure.